src/domain/community/contributor/organization/pages/OrganizationPage.tsx

```
@@ -6,15 +6,26 @@ import Loading from '../../../../../core/ui/loading/Loading';
 import TopLevelLayout from '../../../../../main/ui/layout/TopLevelLayout';
 import { Error404 } from '../../../../../core/pages/Errors/Error404';
 import { useOrganization } from '../hooks/useOrganization';
+import { useOrganizationAccountQuery } from '../../../../../core/apollo/generated/apollo-hooks';
+import useAccountResources from '../../useAccountResources/useAccountResources';
 
 interface OrganizationPageProps {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)